src/views/product-integrations-landing/components/searchable-integrations-list/index.tsx

```diff
@@ -72,28 +100,90 @@ export default function SearchableIntegrationsList({
 	}
 
 	const makeUncheckTierHandler = (e: Tier) => () => {
+		// Set up a tier toggle function that also tracks the tier selection
+		// through an analytics event
+		const tierIndex = tierOptions.findIndex((t) => t === e)
+		const tierToggleFunction = (p) => {
+			const isSelectedNext = !p
+			if (isSelectedNext) {
+				alert(
+					JSON.stringify(
+						{
+							event: 'integration_library_filter_selected',
+							filter_category: 'tier',
+							filter_value: tierIndex,
+						},
+						null,
+						2
+					)
+				)
+				integrationLibraryFilterSelectedEvent({
+					filter_category: 'tier',
+					filter_value: tierIndex,
```

Comment:
I would actually not pass the index here (indexes are subject to change).  I would pass the slug of the filter (e.g. `community`)

Reply:
Ah yep this was me somehow thinking `filter_value` was supposed to be `number` even though, clearly not 😅 have updated!